airflow/www_rbac/views.py

```diff
@@ -988,17 +988,19 @@ def delete(self):
     @expose('/trigger', methods=['POST'])
     @has_dag_access(can_dag_edit=True)
     @has_access
-    @action_logging
     @provide_session
     def trigger(self, session=None):
```

Comment:
Is it not possible to just use decorators here? like for the action logging?

Reply:
i needed the `execution_date` and i don't think there's a way to provide a value to the decorator and then use that in the function body as well